stdlib/2and3/zipfile.pyi

```diff
@@ -22,28 +22,28 @@ class ZipFile:
     debug = ...  # type: int
     comment = ...  # type: bytes
     filelist = ...  # type: List[ZipInfo]
-    def __init__(self, file: Union[str, IO[bytes]], mode: str = ..., compression: int = ...,
+    def __init__(self, file: Union[Text, IO[bytes]], mode: Text = ..., compression: int = ...,
                  allowZip64: bool = ...) -> None: ...
     def __enter__(self) -> ZipFile: ...
     def __exit__(self, exc_type: Optional[Type[BaseException]],
                  exc_val: Optional[Exception],
                  exc_tb: Optional[TracebackType]) -> bool: ...
     def close(self) -> None: ...
-    def getinfo(self, name: str) -> ZipInfo: ...
+    def getinfo(self, name: Text) -> ZipInfo: ...
     def infolist(self) -> List[ZipInfo]: ...
-    def namelist(self) -> List[str]: ...
-    def open(self, name: _SZI, mode: str = ...,
+    def namelist(self) -> List[Text]: ...
```

Comment:
`namelist` can return a list of mixed `unicode` and `str`, which afaict is spelled `List[Text]`:

```python
>>> z = zipfile.ZipFile("new.zip", "w")
>>> z.write(u"spam")
>>> z.write("ni")
>>> z.namelist()
[u'spam', 'ni']
```